### ISSUE TITLE
Update labels: Issue Template: wiki--research-plan-review----replace-with-name-of-page-

### ISSUE THREAD

**Issue by arcan9** [hackforla/website#4488]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] Navigate to `.github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md`
- [x] Replace the following line
   ```
   labels: 'Feature: Wiki, role: user research, size: 0.5pt'
   ```
   with
   ```
   labels: ['Feature: Wiki', 'role: user research', 'size: 0.5pt', 'Complexity: Missing']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository: https://github.com/kiwookim/website/issues/new?assignees=sacamp&labels=Feature%3A+Wiki%2Crole%3A+user+research%2Csize%3A+0.5pt%2CComplexity%3A+Missing&projects=&template=wiki--research-plan-review----replace-with-name-of-page-.md&title=Wiki%3A+Research+Plan+Review%3A++%5BREPLACE+WITH+NAME+OF+PAGE%5D

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md`
- This issue is part of epic #4307

**Comment by ghost** [comment]
For my availability, I should be available the next several days from about 1-8 pm. My ETA on this issue is probably a couple of days because I am still trying to build an understanding of issues and Github.

**Comment by ghost** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
https://github.com/troyfreed/website.git
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: https://github.com/troyfreed/website/issues/1
 This will enable others to check the template.  
- [x] Verify that the labels listed in the issue all appear on the issue template.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [ ] If the labels listed in the issue match those listed on the issue template, complete the PR process by visiting the URL saved from the push command
- [ ] Copy these sections from the Issue into the PR 
  - [ ] "For PR Reviewers and Merge Team"
  - [ ] "Link for Reviewers"
  - [ ] "For PM, Merge Team, or Tech Lead"

**Comment by ExperimentsInHonesty** [comment]
@troyfreed I am removing you from this issue.   It was too big for a good first issue, and we want to set you up with an appropriate size issue.

**Comment by kiwookim** [comment]
Availability: 9-9pm
ETA: 7/10 EOD

**Comment by kiwookim** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
https://github.com/kiwookim/website/pull/new/update-labels-issue-template-4488
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: 
https://github.com/kiwookim/website/issues/new?assignees=sacamp&labels=Feature%3A+Wiki%2Crole%3A+user+research%2Csize%3A+0.5pt%2CComplexity%3A+Missing&projects=&template=wiki--research-plan-review----replace-with-name-of-page-.md&title=Wiki%3A+Research+Plan+Review%3A++%5BREPLACE+WITH+NAME+OF+PAGE%5D
 This will enable others to check the template.  
- [x] Verify the appearance of the issue template, including careful examination of the labels.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [x] Complete the PR process by visiting the URL saved from the push command
- [x] Copy these sections from the Issue into the PR 
  - [x] "For PR Reviewers and Merge Team"
  - [x] "Link for Reviewers"
  - [x] "For PM, Merge Team, or Tech Lead"
